packages/api-explorer/__tests__/JsonForm/__snapshots__/index.test.jsx.snap

```diff
@@ -44,13 +44,14 @@ exports[`JSONForm  snapshot 1`] = `
     type="submit"
   />
   <div
-    class="je-object__container mia-container-wrapper"
```

Comment:
What do you think causes these `je-` classes to disappear? 🤔  have you updated some dependency in you dev environment?

Reply:
It looks strange also to me. I simply run `npm install` then `npm run boot` following the README. After that I have modified some `package-lock.json`. And then the snapshot is changed.

Reply:
you should use `npm ci`, not `npm install`, so that you install exactly what is specified in the package-lock

Reply:
I remove the updated snapshot. Btw I am still getting the snapshot `not matching` error after removing node_modules and running `npm ci`

Reply:
not matching because the classes are still removed?

Reply:
yes, I get the same error also in master. So I update that snapshot.